layouts/partials/scientific-domains.html

```diff
@@ -70,28 +57,17 @@
             <td class="lastrow-center-text"></td>
             <td class="lastrow-center-text"></td>
         </tr>
+
         <!-- Second Row -->
         <tr class="highlight-th">
-            <td class="bold-text">Bioinformatics</td>
-            <td class="bold-text">Bayesian Inference</td>
-            <td class="bold-text">Mathematical Analysis</td>
-            <td class="bold-text">Chemistry</td>
-            <td class="bold-text">Geoscience</td>
-            <td class="bold-text">Geographic Processing</td>
-            <td class="bold-text">Architecture & Engineering</td>
+            {{- range $librariesrow2 }}
+                <td class="bold-text">{{ .title }}</td>
+            {{- end }}
         </tr>
         <tr>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/bioinformatics.svg"
-                    alt="A strand of DNA."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/bayesian_inference.svg"
-                    alt="A graph with a bell-shaped curve."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/mathematical_analysis.svg"
-                    alt="Four mathematical symbols."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/chemistry.svg" alt="A test tube."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/geoscience.svg" alt="The Earth."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/GIS.svg" alt="A map."></td>
-            <td><img class="cell-layout" src="/images/content_images/sc_dom_img/robotics.svg"
-                    alt="A microprocessor development board."></td>
+            {{- range $librariesrow2 }}
+                <td><img class="cell-layout" src="{{ .img }}" alt="{{ .alttext }}"></td>
+            {{- end }}
         </tr>
         <tr>
             <td class="center-text"><a href="https://biopython.org">BioPython</a></td>
```

Comment:
Do we need to shift these links to YAML files? I am asking because, I think even after translation to any language they will remain the same. The case with NumPy is the same and there is just link and library name here. In that case, it won't be much beneficial to have the same content repeated (in English) in all the yaml files under folders of different languages.

Reply:
Agreed, can stay here - names and links should not be translated